src/rows.jl

```diff
@@ -119,7 +119,7 @@ function Base.showerror(io::IO, e::UnknownSchemaError)
               your environment is as expected).
 
               Note that if you're in this particular situation, you can still load
-              the raw Arrow table as-is via `Arrow.Table`.
+              the raw table as-is. For example, to load an Arrow table, call `Arrow.Table(path)`.
```

Comment:
```suggestion
              the raw table as-is without Legolas; e.g., to load an Arrow table, call `Arrow.Table(path)`.
```

slightly clearer/more helpful for users who might not understand what `as-is` means